patches/components-vector_icons-vector_icons.gni.patch

```diff
@@ -1,21 +1,21 @@
 diff --git a/components/vector_icons/vector_icons.gni b/components/vector_icons/vector_icons.gni
-index d068c55a04ac027af5c3c3c9be8e13df4df5f09a..25ca9fabf28ee55b52dfeee3b4330fa5bfa687f7 100644
+index d068c55a04ac027af5c3c3c9be8e13df4df5f09a..47609044402cde99c4beceb8a97c234bd11be2f9 100644
 --- a/components/vector_icons/vector_icons.gni
 +++ b/components/vector_icons/vector_icons.gni
-@@ -60,6 +60,7 @@ template("aggregate_vector_icons2") {
-       "--file_list={{response_file_name}}",
-       "--output_cc=" + rebase_path(output_cc, root_build_dir),
-       "--output_h=" + rebase_path(output_h, root_build_dir),
-+      "--alt_working_directory=" + alt_icons_dir_rel,
-     ]
-   }
- }
-@@ -84,6 +85,8 @@ template("aggregate_vector_icons") {
+@@ -42,6 +42,8 @@ template("aggregate_vector_icons2") {
        "vector_icons.cc.template",
        "vector_icons.h.template",
      ]
 +    alt_icons_dir = "//brave/vector_icons/"+ rebase_path(invoker.icon_directory, "//")
```

Comment:
this should be on one line

Reply:
Fixed in 9fb846e08d897ab0ae79be007aaee025e2d5667c